docs/collector.mssql.md

```diff
@@ -249,7 +253,16 @@ Name | Description | Type | Labels
 _This collector does not yet have explained examples, we would appreciate your help adding them!_
 
 ## Useful queries
-_This collector does not yet have any useful queries added, we would appreciate your help adding them!_
+
+### Buffer Cache Hit Ratio
+
+When you read the counter in perfmon you will get the the percentage pages found in the buffer cache. This percentage is calculated internally based on the total number of cache hits divided by the total number of cache lookups over the last few thousand page accesses.
+This collector retrieves the two internal values separately. In order to give any meaning to the value you will have to calculate the Buffer Cache Hit Ratio in PromQL.

```

Comment:
This is a bit strongly formulated - the values *do* have meaning separately. Just not the hit ratio, which the current naming implies. So with improved naming in the PR, I think this will be less confusing.
The example query is still good to have here, though!